libbpf-tools/trace_helpers.c

```diff
@@ -552,7 +553,8 @@ static int create_tmp_vdso_image(struct dso *dso)
 		return -1;
 
 	while (true) {
-		ret = fscanf(f, "%lx-%lx %*s %*x %*x:%*x %*u%[^\n]",
+		ret = fscanf(f,
+			     "%" PRIx64 "-%" PRIx64 "%*s %*x %*x:%*x %*u%[^\n]",
```

Comment:
I really dislike those PRIxxx macros, in libbpf we just explicitly cast to (long long) and use %lld, can you please do the same here?

Reply:
I don't love them either but these are used in several other source files in this code base. Shouldn't we keep things consistent here?

Reply:
where do you see it used in libbpf-tools tools?

Reply:
I meant in the root project, bcc.